data/json/monsters/bird.json

```diff
@@ -194,6 +198,7 @@
   },
   {
     "id": "mon_chicken_chick",
+    "looks_like": "mon_chicken",
```

Comment:
There is the same string on line 216 already.

```suggestion
```

Reply:
Removed Line 216's for consistent formatting.
(Finals began this week, I'm not dead on this yet 😛 )